### PR TITLE
URL parameter translation

### PR DESCRIPTION
**Pull request by superyusuke**
Almost done without one question , leaving in text.
Please check it.

### CODE REVIEW

File: src/content/guides/url-parameters/document.ja.mdx [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+      <NotFoundBoundary render={() => <h1>Quote Not Found</h1>}>
+        <View />
+      </NotFoundBoundary>
+    </Suspense>
+  </Router>,
+  document.getElementById('root')
+)
+```
+
+URL parameters は JavaScript ルーティングライブラリにおいて非常に標準的な機能ではありますが、URL parameters と asynchronous content を組み合わせることで、莫大な宣言的な力を手に入れることができます。MVC フレームワークにおけるコントローラーのように Navi の route 機能は、ビジネスロジックの実装し、エラーを投げ、View を選択することができます。-- all based on the request. (nakanishi-question: what this request mean? HTTP request?)
```
Comment (jamesknelson) [inline src/content/guides/url-parameters/document.ja.mdx]: In this case, "Based on the request" means based on the `req` object (which is short for `request`). Perhaps "based on the request object passed to the route function" would be clearer?

Comment (superyusuke) [reply src/content/guides/url-parameters/document.ja.mdx]: Thanks! I understand you!

**Comment by superyusuke** [comment]
All done !!

**Comment by jamesknelson** [comment]
Thank you! I will publish this today.

**Comment by superyusuke** [comment]
Thanks!

**Pull request merged**